### PR TITLE
Vlada dev

### PR DESCRIPTION
**Pull request by solo-vlada**

### CODE REVIEW

File: social app/js/requests.js [hunk 1]
```diff
@@ -10,35 +10,43 @@ async function getPost(title, date) {
...
         }
 
         const response = await fetch('http://localhost:3000/posts', options);
-        const {id, err} = await response.json;
-        if(err) { 
-            throw Error(err)
-        } else {
-            window.location.hash = `${title}/${date}`
-        }
+        const data = await response.json;
```
Comment (brhanuh) [inline social app/js/requests.js:31]: May need parenthesis here -> 'response.json()'

**Review by brhanuh — CHANGES_REQUESTED** [review]

**Review by mariesophiecd — COMMENTED** [review]
We couldn't access all the files from the review. So we looked into your repo.
In your main.js, you have a function to validate your form. We were thinking with Hanibal that you can also add "required" to your inputs in the html. 

You did a great work by yourself in a very short amount of time!

**Comment by brhanuh** [comment]
I couldn't see all the changes to the repo because they were pushed before the pull request. I can only see the readme and request.js file (with postNewPost method). It would have been better if you made a pull request to a different branch with no previous commits. But other than that great work for the short time you had (:

File: social app/js/requests.js (hunk 1 above)
Comment (solo-vlada) [reply social app/js/requests.js:31]: thanks! will look into adding them!

**Pull request merged**